src/lib/style/components/_user-profile-links.scss

```diff
@@ -0,0 +1,28 @@
+@use '../abstracts/mixins' as SRCMixins;
+@use '../abstracts/variables' as SRC;
+
+.UserProfileLinks {
+  // box shadow on the left hand side only.
+  // horizontal offset -12px, vertical offset 0px, and reduce the shadow spread with -15px
+  box-shadow: -12px 0px 20px -15px rgb(74 80 86 / 15%);
+  width: 365px;
+  .title {
+    padding: 30px 0px 20px 30px;
+  }
+  .Tabs {
+    @include SRCMixins.underline-tabs;
+    .Tab {
+      padding-left: 10px;
+    }
+  }
+
+  .TabContent {
+    min-height: calc(100vh - 100px);
+    max-height: calc(100vh - 100px);;
```

Comment:
Just curious: what are these doing?

Reply:
It's to render a potentially long list of projects/teams/challenges in a scroll panel, constrained to the height of the viewport (so you could scroll outside of this area and still access the footer, for example).